serverless.yaml

```diff
@@ -5,6 +5,10 @@
 
 service: fhir-service
 
+package:
+  include:
+    - bulkExport/glueScripts/export-script.py

```

Comment:
It might be worth moving `bulkExport/glueScripts/export-script.py` to `bulkExport/export-script.py`. I think we put the script in a folder before because we thought we have to upload everything in a folder to S3. However, if we can select specific files, putting the script in it's own folder seems redundant.

Reply:
I'll keep it as is. There's no harm in having an extra folder, plus I like how it separates the `.py` files from the rest.